### PR TITLE
replace ember-cli-release with release it

### PR DESCRIPTION
**Pull request by nvdk**
as a side effect, this removes plain css support for now

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -118,11 +116,18 @@
...
     "last 1 Firefox version",
-    "last 1 FirefoxAndroid version",
-    "last 1 Safari version",
-    "last 1 iOS version",
     "last 1 Edge version"
-  ]
+  ],
+  "release-it": {
+    "github": {
+      "release": false
```
Comment (Windvis) [inline package.json:124]: I think `false` is the default so this can be removed if this isn't a wanted feature.

File: index.js [hunk 2]
```diff
@@ -9,16 +9,6 @@ module.exports = {
   },
 
   included() {
```
Comment (Windvis) [inline index.js:11]: The whole method can be removed I think?

File: package.json [hunk 3]
```diff
@@ -118,11 +116,18 @@
...
+    },
+    "npm": {
+      "publish": false
+    },
+    "git": {
+      "requireCleanWorkingDir": true
```
Comment (Windvis) [inline package.json:130]: I think this is also the default behavior? https://github.com/release-it/release-it/blob/master/docs/git.md#clean-working-directory

**Review by Windvis — APPROVED** [review]
Only some small / nitpicky things, so feel free to ignore that :smile:.

File: index.js (hunk 2 above)
Comment (nvdk) [reply index.js:11]: I've kept it, because last time something was added to included we forgot to call super. but you're right it can go.

**Pull request merged**